PlainTasks.py

```diff
@@ -28,6 +28,14 @@
     import subprocess
 
 
+def tznow():
+    return datetime.now().replace(
```

Comment:
How it differs from `datetime.now(tz=timezone(datetime.fromtimestamp(0) - datetime.utcfromtimestamp(0)))`?